### DISCUSSION TITLE
No response from GitHub Support for 35 days

### DISCUSSION THREAD

**Discussion by nodeMevK** [community/community#134090] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

It has officially been 35 days since I have submitted my ticket and have not received any response from github support. My account [https://github.com/ghostMevK](url) was flagged without notice or explanation. 

I've tried reaching out to the team on other social medias and still no response.

I'm not sure if this is a regular wait time when coming to this but would love to get a response to my ticket.

Thank you.

**Comment by azflin** [comment]
Devs help a brother out here pls

**Comment by Sivamani-18** [comment]
I understand your frustration with the delayed response from GitHub support. While 35 days is indeed a lengthy wait time, there are a few steps you can take to potentially expedite the process:

1. **Follow Up on Your Ticket:** Make sure to reply to your existing support ticket to follow up. This can help bring attention to your issue.

2. **Contact GitHub Support via Twitter:** Tweeting directly to GitHub Support (@GitHubHelp) on Twitter might help in getting a quicker response.

3. **Check GitHub's Status Page:** Occasionally, delays in support responses can be due to ongoing issues or higher-than-normal support volumes. You can check [GitHub's status page](https://www.githubstatus.com/) for any updates or announcements.

4. **Community Support:** Posting about your issue on GitHub Community forums or relevant Reddit threads might attract the attention of GitHub staff or experienced community members who can offer advice.

5. **Email Support:** If you haven't already, sending a follow-up email to GitHub Support (support@github.com) might help.

6. **Documentation Review:** Review GitHub’s [community guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines) and [terms of service](https://docs.github.com/en/github/site-policy/github-terms-of-service) to ensure there hasn’t been a misunderstanding that led to your account being flagged.

**Comment by TresFlames** [comment]
This is a wrongful termination of the account. Any conflicts can surely be resolved. As a product of this I can imagine @nodeMevK has lost countless development cycles and productivity to his ongoing projects. What a cumbersome move from GitHub. 

Please aid in reviving the ghost.

**Comment by owencmcgrath** [comment]
My account was also flagged without any explanation!

I've been waiting for about a week now. 

Here's hoping we both get this resolved soon.

**Comment by wiizzoux** [comment]
> ### Select Topic Area
> 
> General
> ### Body
> 
> Hello,
> 
> It has officially been 35 days since I have submitted my ticket and have not received any response from github support. My account [https://github.com/ghostMevK](url) was flagged without notice or explanation.
> 
> I've tried reaching out to the team on other social medias and still no response.
> 
> I'm not sure if this is a regular wait time when coming to this but would love to get a response to my ticket.
> 
> Thank you.

Same problem. No response since May 7th (88 days!)

**Reply by owencmcgrath** [reply]
> > ### Select Topic Area
> 
> > 
> 
> > General
> 
> > ### Body
> 
> > 
> 
> > Hello,
> 
> > 
> 
> > It has officially been 35 days since I have submitted my ticket and have not received any response from github support. My account [https://github.com/ghostMevK](url) was flagged without notice or explanation.
> 
> > 
> 
> > I've tried reaching out to the team on other social medias and still no response.
> 
> > 
> 
> > I'm not sure if this is a regular wait time when coming to this but would love to get a response to my ticket.
> 
> > 
> 
> > Thank you.
> 
> 
> 
> Same problem. No response since May 7th (88 days!)

That is insanely frustrating, I'm sorry to hear that.

Have you not heard anything back at all?

**Comment by SAHELISAHAA** [comment]
@nodeMevK lets make tweet together

**Reply by Budalebah** [reply]
have you fixed it ?

**Comment by ghost** [comment]
Did you solve it?
I created a ticket regarding account recovery, but there has been no response for 7 days. I am very anxious.

**Reply by haloxtradee** [reply]
No i havent. Still waiting... and seems like we will wait more and more

**Comment by nodeMevK** [comment]
Still waiting for a response from support lol. Been about 3 months now.  Ridiculous really
